examples/rust/profile/src/main.rs

```diff
@@ -224,6 +239,9 @@ struct Args {
     /// Increase verbosity (can be supplied multiple times).
     #[arg(short = 'v', long = "verbose", global = true, action = ArgAction::Count)]
     verbosity: u8,
+    /// Software event
+    #[arg(long = "sw-event", action = ArgAction::SetTrue)]
```

Comment:
Haven't tried it, but my understanding is that the default action for `bool`s is to set it to true, so you can probably omit the `action = ...` part here.

Reply:
Tried. The action part indeed don't need here, it's the default behavior.